scripts/west_commands/runners/jlink.py

```diff
@@ -208,6 +209,8 @@ def do_add_parser(cls, parser):
         parser.add_argument('--dev-id-type', choices=['auto', 'serialno', 'tty', 'ip', 'tunnel'],
                             default='auto', help='Device type. "auto" (default) auto-detects '
                             'the type, or specify explicitly')
+        parser.add_argument('--dp-reset-off', default=False, action='store_true',
+                            help='if given, disables DP reset')
```

Comment:
Prefer to use the `ToggleAction` here:

```suggestion
        parser.add_argument('--dp-reset', '--no-dp-reset', nargs=0,
                            action=ToggleAction,
                            help='reset Debug Port registers? (default: yes)')
```